modules/apps/portal/portal-cluster-multiple/src/main/java/com/liferay/portal/cluster/multiple/internal/ClusterExecutorImpl.java

```diff
@@ -264,6 +264,8 @@ protected ClusterNodeResponse executeClusterRequest(
 						" that is not serializable")));
 		}
 		catch (Exception exception) {
+			_log.error("Unable to process cluster request", exception);
```

Comment:
This is a bad change, especially for cloud usecase.
When an error happens, we have a choice of either log it or propagate it back to invoker.

**But we should NOT do both.**

LXC has complained about large amount of log eats up disk space. If we do both logging and propagation, how do you know the invoker would not log it again at his level? If this type of design is spread across system, we could end up logging the same info multiple times from different level of invokers.

If you see an error info is missing, trace it to the invoker, it is one of the invoker ignoring the Exception info in the ClusterNodeResponse. Fix it at the invoker level, rather than blindly log it inside the framework.

Reply:
SaaS complains about the large amount of log (since that's money) but on-premise customers claim for the opposite, they do not have enough information to address issues by their own.

I agree the ideal solution is to fix the issue in the invoker but my understanding was to add that log trace to get more information about hidden clustering issues we are experiencing now. Then, fix in the invoker having the trace information.

Anyways, let's analyze it in detail.